src/sentry/monitors/models.py

```diff
@@ -370,9 +370,11 @@ def update_config(self, config_payload, validated_config):
     def get_validated_config(self):
         try:
             jsonschema.validate(self.config, MONITOR_CONFIG)
-            return self.config
         except jsonschema.ValidationError:
-            logging.exception("Monitor: %s invalid config: %s", self.id, self.config)
+            logging.warning("Monitor: %s invalid config: %s", self.id, self.config, exc_info=True)
+        # We should always return the config here - just log an error if we detect that it doesn't
+        # match the schema
```

Comment:
To some extent I wonder if this validation is even useful at this step... I guess it's good to know if somehow invalid configs are getting into the system so we can fix them